meson.build

```diff
@@ -44,6 +44,7 @@ dftables = executable('dftables',
   'dftables.c',
   c_args : '-DHAVE_CONFIG_H',
   include_directories : conf_inc,
+  native : true,
```

Comment:
Could you please add a comment here, something like

```
  # http://mesonbuild.com/Cross-compilation.html#mixing-host-and-build-targets
  # This target is used only during the build.
  native : true,
```

Or whatever wording you prefer. This option is not obvious for people not working a lot with cross builds and is missing in the [reference manual](http://mesonbuild.com/Reference-manual.html) so I believe this is a good idea to document it here.